modules/types/src/channel.ts

```diff
@@ -0,0 +1,87 @@
+import { Node as CFCoreTypes } from "@counterfactual/types";
+import { BigNumber } from "ethers/utils";
+
+import { App, AppState } from "./app";
+import { Address } from "./basic";
+
+////////////////////////////////////
+////// CHANNEL TYPES
+
+// used to verify channel is in sequence
+export type ChannelAppSequences = {
+  userAppSequenceNumber: number;
+  nodeAppSequenceNumber: number;
```

Comment:
we should just call these `userSequenceNumber` rather than `userAppSequenceNumber`